flexget/plugins/search/1337x.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import unicode_literals, division, absolute_import
+from future.moves.urllib.parse import quote
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+import re
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.requests import TimedLimiter, RequestException
```

Comment:
Please go through all your imports and make sure they are used. You are not using TimedLimiter.